tools/sim/launch_openpilot.sh

```diff
@@ -6,7 +6,7 @@ export SIMULATION="1"
 export SKIP_FW_QUERY="1"
 export FINGERPRINT="HONDA_CIVIC_2022"
 
-export BLOCK="${BLOCK},camerad,loggerd,encoderd,micd,logmessaged"
+export BLOCK="${BLOCK},camerad,loggerd,encoderd,micd,logmessaged,logcatd,proclogd"
```

Comment:
manager should just know not to run these on macOS. can you do this in pocess_config.py instead?